admin/admin-model.php

```diff
@@ -454,6 +454,20 @@ public function update_translations( $old_slug, $new_slug = '' ) {
 			foreach ( $terms as $term ) {
 				$term_ids[ $term->taxonomy ][] = $term->term_id;
 				$tr = maybe_unserialize( $term->description );
+
+				/**
+				 * Filters the unserialized translation group description before it is
+				 * updated when a language is deleted or a language slug is changed.
+				 *
+				 * @since 3.2
+				 *
+				 * @param array   $tr       The translations group.
```

Comment:
What about using something similar to your other PR?

```php
				 * @param array<int|array<string>> $tr       {
				 *     List of translations with lang codes as array keys and IDs as array values.
				 *     Also in this array:
				 *
				 *     @type array<string> $sync List of synchronized translations with lang codes as array keys and array values.
				 * }
```

Reply:
Good idea :+1: